collect_app/src/main/res/values/settings_theme.xml

```diff
@@ -2,12 +2,14 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
 
     <style name="Theme.Collect.Settings.Light" parent="Theme.Collect.BaseLight">
+        <item name="android:textColor">?primaryTextColor</item>
```

Comment:
We're starting to move towards [Material Theming colors](https://material.io/develop/android/theming/color/) so it would be best to use `?colorOnSurface` here instead. Totally understand it's confusing to have both still defined in the app.

Reply:
Alright! I changed it.